docs/docs/cmd/tenant/serviceannouncement/serviceannouncement-health-get.md

```diff
@@ -118,3 +118,20 @@ When we make use of the option `issues` the response will differ.
     id,status,service
     Exchange,serviceDegradation,Exchange Online
     ```
+
+=== "Markdown"
+
+    ```md
+    # tenant serviceannouncement health get --serviceName "Exchange Online"
+
+    Date: 4/19/2023
+
+    ## Exchange
+
+    Property | Value
+    ---------|-------
+    service | Exchange Online
+    status | serviceDegradation
+    id | Exchange
+    ```
```

Comment:
This response should be moved a bit above to "standard response" and we should add another response on this place using the `--issues` flag.